vaadin-grid-flow-parent/vaadin-grid-testbench/src/main/java/com/vaadin/flow/component/grid/testbench/GridElement.java

```diff
@@ -388,7 +388,7 @@ void select(GridTRElement row) {
             CheckboxElement checkbox = wrapElement(cell.getFirstChildElement(),
                     getCommandExecutor()).wrap(CheckboxElement.class);
             if (!checkbox.isChecked()) {
-                checkbox.click();
+                checkbox.getWrappedElement().click();
```

Comment:
In TestBench, the `click()` method is overridden to use the JS `click()` function, which doesn't fully simulate a complete mouse action. We need to use the real simulation to get `mousedown` that sets `_shiftKeyDown`.